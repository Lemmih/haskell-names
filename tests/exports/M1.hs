module M1 where

a = a
