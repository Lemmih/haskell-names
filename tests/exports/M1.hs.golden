Symbols (
  fromList
    [ SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M1"
                                          "a"
                          }
        , sv_fixity = Nothing
        }
    ] ) (
  fromList [] )